README.md

```diff
@@ -18,7 +18,7 @@ Add the library from [crates.io](https://crates.io/crates/extism-pdk).
 cargo add extism-pdk
 ```
 
-Change your `Cargo.toml` to set the crate-type to `cdylib`:
+Change your `Cargo.toml` to set the crate-type to `cdylib` (this instructs the compiler to produce a Wasm file):
```

Comment:
This isn't exactly accurate. Compiling to a wasm32-* target is what makes the output a wasm file. This tells the compiler that you are emitting a dynamic library and not an executable (which in wasm parlance means a command pattern module with a _start function)

Reply:
How about this:
```
Change your `Cargo.toml` to set the crate-type to `cdylib` (this instructs the compiler to produce a dynamic library, which for our target will be a Wasm file):
```

Reply:
If you're writing in c or rust your "main" function is called `main` in the source, but when it turns into wasm it becomes can export called `_start`

Reply:
>  I meant as opposed to a .dylib or .so

That would be determined by the target flag. This is from the wasm-pack doc but relevant:

```
Here though crate-type = ["cdylib"] typically signifies that you'd like the compiler to create a dynamic system library, but for WebAssembly target it simply means "create a *.wasm file without a start function". On other platforms this output type will create *.so file on Linux, *.dylib on macOS, and *.dll Windows.
```

Reply:
:+1: I think that definition is in line with my latest update